cli/flags.rs

```diff
@@ -2524,10 +2542,10 @@ mod tests {
   #[test]
   fn upgrade() {
     let r = flags_from_vec(svec!["deno", "upgrade", "--dry-run", "--force"]);
-    let flags = r.unwrap();
+    let inner = Arc::try_unwrap(r.unwrap().0).unwrap();
     assert_eq!(
-      flags,
-      Flags {
+      inner,
```

Comment:
can you not do just `*r.unwrap()` ?

Reply:
Moving out  of an `Arc` is imposible, but I can replace it by somethink like that:

``
r.unwrap().0.as_ref().clone();
``

Reply:
If you are the only holder of the `Arc` then you can use `try_unwrap` to move out of the arc: https://doc.rust-lang.org/std/sync/struct.Arc.html#method.try_unwrap

Reply:
> If you are the only holder of the `Arc` then you can use `try_unwrap` to move out of the arc: https://doc.rust-lang.org/std/sync/struct.Arc.html#method.try_unwrap

and I did it at line 2545:
`let inner = Arc::try_unwrap(r.unwrap().0).unwrap();`

@bartlomieju there something I'm missing here?

Reply:
Ah didn't notice. Seems okay 👍